osu.Game/Overlays/SkinEditor/SkinEditor.cs

```diff
@@ -256,13 +256,13 @@ private void skinChanged()
 
             headerText.AddParagraph(SkinEditorStrings.SkinEditor, cp => cp.Font = OsuFont.Default.With(size: 16));
             headerText.NewParagraph();
-            headerText.AddText("Currently editing ", cp =>
+            headerText.AddText(SkinEditorStrings.CurrentlyEditing, cp =>
```

Comment:
This may not work great with all languages, but I'm not sure we can do better currently if we want the formatting to remain as it is here.